README.md

```diff
@@ -46,6 +46,38 @@ fact, without the
 ```
 that's the version that will be used by default.
 
+# Quickstart - Maven Example
```

Comment:
First off, I guess I should have really pointed at our "[Using ESAPI with Jakarta EE Servlet API Specification 5.0 and later](https://github.com/ESAPI/esapi-java-legacy/wiki/Using-ESAPI-with-Jakarta-EE-Servlet-API-Specification-5.0-and-later)" wiki page instead. Pretty much all that you need to know is over there about this is there, and if it isn't, that's where it should be placed, not here. (Unfortunately, only the core team can make PRs for that; it seems to be a GitHub thing.) I just checked, and our current README file is already 8 pages. If anything, I'd like to make it shorter and more some of the nonessential content to our wiki page. The only reason that I added a section for "Jakarta EE Support" is because not too long ago I was getting slammed with personal emails asking me "does ESAPI support Jakarta EE yet". Adding that cut down on that, but I definitely should have linked to the corresponding wiki page from here.

So, look at the wiki page that I've referenced and if you think that the wiki page needs updated, please create a GitHub issue outlining the suggested change and ideally include a "patch" file that I can apply to update it and we can handle it that way (since you can accept a PR against the wiki pages.)

Reply:
@DebajitKumarPhukan - I have a bunch of changes (mostly reorganization and simplification) to our README.md, but I was waiting until you finished this PR so we wouldn't get merge conflicts. I may still move that content to our wiki page, or at least change it and drop it into a collapsible section. But our README is already just a shade over 8 pages if you were to print it. I think that's about 3 or 4 pages too long. But this is good enough for now.